distribution/docs/src/main/jdocs/content/_security-framework/security-token-service-contents.adoc

```diff
@@ -0,0 +1,28 @@
+:title: Security Token Service
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:children: Security PEP Interceptor
+:order: 08
+:summary: Security Token Service.
+
+The Security Token Service (STS) is a service running in ${branding} that generates SAML v2.0 assertions.
+These assertions are then used to authenticate a client allowing them to issue other requests, such as ingests or queries to ${branding} services.
+
+The STS is an extension of Apache CXF-STS.
+It is a SOAP web service that utilizes WS-Trust.
+The generated SAML assertions contain attributes about a user and is used by the Policy Enforcement Point (PEP) in the secure endpoints.
+Specific configuration details on the bundles that come with ${branding} can be found on the Security STS application page.
+This page details all of the STS components that come out of the box with ${branding}, along with configuration options, installation help, and which services they import and export.
+
+The STS server contains validators, claim handlers, and token issuers to process incoming requests.
+When a request is received, the validators first ensure that it is valid.
+The validators verifies authentication against configured services, such as LDAP, DIAS, PKI.
+If the request is found to be invalid, the process ends and an error is returned.
+Next, the claims handlers determine how to handle the request, adding user attributes or properties as configured.
+The token issuer creates a SAML 2.0 assertion and associates it with the subject.
+The STS server sends an assertion back to the requestor, which is used in both SOAP and REST cases.
```

Comment:
" which is used in both SOAP and REST cases." --->
"which is used to authenticate and authorize subsequent SOAP and REST requests"

Reply:
updated.